terraform/vars.tf

```diff
@@ -50,6 +50,13 @@ variable "cloudsql_disk_size_gb" {
   description = "Size of the Cloud SQL disk, in GB."
 }
 
+variable "create_generate_service" {
```

Comment:
If we went the "always create and deploy this service, but never actually run it unless the user configures it" route, this would be something like `generate_schedule_cron` instead.